server/src/main.ts

```diff
@@ -25,7 +25,7 @@ const sessionManager = new SessionManager(options.verbose || false);
 httpServer.post("/get_pot", async (request, response) => {
     const visitorData = request.body.visitor_data as string;
     const dataSyncId = request.body.data_sync_id as string;
-
+    const proxies: string[] = Object.values(request.body.proxies);
```

Comment:
do we need this?
we've already processed them with `'proxies': self.proxies.values(),`